script/setup

```diff
@@ -15,16 +15,17 @@ if [ ! -f "$SETTINGS_FILE" ]; then
 fi
 
 mkdir -p config
+python3 -m pip install uv
```

Comment:
Should this install a specific version? We already have a version of uv set in the HA dependencies

Reply:
I was thinking that we should use the latest version in the devcontainer by default but I'm happen to use the same version as specified in the requirements

Reply:
This command will fail when the developer is using a Linux distro which uses their system package manager like `apt`, `pacman` etc. for python packages.

We should check if uv us available and if not create the venv with standard python venv and inside the venv install uv and then switch to it.

Reply:
And we need to install uv inside the venv as well as the first thing after the venv creation, because system-installed packages may be not available inside the venv.

Reply:
I would rather do this:

```bash
if [ -x "$(command -v uv)" ]; then
    VENV_COMMAND="uv venv venv"
else
    VENV_COMMAND="python3 -m venv venv"
fi

if [ ! -n "$VIRTUAL_ENV" ]; then
  $VENV_COMMAND
  source venv/bin/activate
fi

python3 -m pip install uv

script/bootstrap
```

So we can be happy when uv is already installed and when it is not installed just create the venv with normal python and then install uv inside.

Reply:
Why should we install `uv` again if it is already installed?

Reply:
We can put the installation of uv into an if as well.